spec/index.bs

```diff
@@ -1249,17 +1249,9 @@ partial interface mixin WindowOrWorkerGlobalScope {
 </pre>
 
 To the [[HTML5#timer-initialisation-steps|timer initialization steps algorithm]],
-add this step between 7.1 and 7.2:
+change step 8.3.2 as follows:
 
-1.  If the first operation argument is not a {{Function}}, set the first operation argument to the result of executing
-    the [$Get Trusted Type compliant string$] algorithm, with
-    *   |global| set to the [=this=] value's [=relevant global object=].
-    *   |input| set to the first method argument, and
-    *   |expectedType| set to {{TrustedScript}}.
-    *   |sink| set to `Window setInterval` if <var ignore>repeat</var> is true, `Window setTimeout` otherwise.
-    *   |sinkGroup| set to `'script'`.
-
-        Note: This matches the logic that the extended attribute would apply.
+1.  Perform HostEnsureCanCompileStrings(<var>realm</var>, « », <var>handler</var>,<del> false</del><ins>, <var>handler</var>, ~timer~, « », <var>handler</var></ins>). If this throws an exception, catch it, report the exception, and abort these steps.
```

Comment:
Why isn't `HostEnsureCanCompileStrings` patched first? It seems that'd be cleaner. https://html.spec.whatwg.org/#timer-initialisation-steps takes different arguments than the ones passed here.

Reply:
HostEnsureCanCompileStrings will be updated accordingly, don't think it's worth adding it in this PR. https://w3c.github.io/trusted-types/dist/spec/#csp-eval already updates EnsureCSPDoesNotBlockStringCompilation so it's really just the HTML HostEnsureCanCompileStrings block that isn't handled.

Once the corresponding tc39 change gets merged it'll make the layering of all these changes much easier.

Reply:
https://github.com/w3c/trusted-types/pull/490 - made a PR to update the host function in HTML land.